src/tasks/export-data/export-data-v2/task.json

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "3137A4C5-0E44-4D89-AC0C-BEEDEC036428",
+  "name": "PowerPlatformExportData",
+  "friendlyName": "Power Platform Export Data",
+  "description": "Power Platform export data from environment with provided schema",
+  "author": "Microsoft",
+  "helpMarkDown": "[More Info](https://aka.ms/buildtoolsdoc)",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "PowerPlatformTool"
+  ],
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 0
+  },
+  "preview": false,
+  "minimumAgentVersion": "1.95.0",
+  "instanceNameFormat": "Power Platform Export Data $(message)",
+  "inputs": [
+    {
+      "name": "authenticationType",
+      "type": "radio",
+      "label": "Authentication type",
+      "defaultValue": "PowerPlatformEnvironment",
+      "required": true,
+      "helpMarkDown": "Authentication via username/password is simpler to set up but won't work if your AAD tenant requires MFA; use Service Principal Name (SPN) if MFA is required.",
+      "options": {
+        "PowerPlatformEnvironment": "Username/password (no MFA support)",
+        "PowerPlatformSPN": "Service Principal/client secret (supports MFA)"
+      }
+    },
+    {
+      "name": "PowerPlatformEnvironment",
+      "type": "connectedService:Generic",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformEnvironment",
+      "helpMarkDown": "Authenticate with your Power Platform environment with username/password. Does not support MFA."
+    },
+    {
+      "name": "PowerPlatformSPN",
+      "type": "connectedService:powerplatform-spn",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformSPN",
+      "helpMarkDown": "Authenticate with your Power Platform environment with an Azure AppID, tenantID and client secret."
+    },
+    {
+      "name": "SchemaFile",
+      "label": "Schema File",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "Schema file name. It can be created using Configuration Migration Tool."
+    },
+    {
+      "name": "DataFile",
+      "label": "Data File",
+      "type": "string",
+      "defaultValue": "data.zip",
+      "required": true,
+      "helpMarkDown": "File name for data zip file. Default data.zip."
+    },
+    {
+      "name": "Overwrite",
+      "label": "Overwrite",
+      "type": "boolean",
+      "required": false,
+      "helpMarkDown": "Allow overwrite output data file if it already exists."
+    },
+    {
```

Comment:
please use the same text and default values as for other instance tasks. The order in this .json matters, it should come right after the svc conn entries to stay consistent w/ the task GUI.
e.g. see #189